scripts/lisk/docker/setEnvVariables.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+# Set env var from secrets
+# secret_id=arn:aws:secretsmanager:eu-west-3:132202091885:secret:mainnet/lisk-across-relayer/aws-CSi7ka
+secret_id=arn:aws:secretsmanager:eu-west-3:132202091885:secret:sepolia/across-relayer-dev/aws-7CIqpl
```

Comment:
What about here? Should we use `mainnet` instead of `sepolia`?

Reply:
Yes, as mentioned earlier on Slack, I'll finally update this and the workflow file in the final commit before merging the PR.
Setting it to `mainnet` currently will run another competing instance of the bot on the mainnet :wink: